source/administration/upgrade-script.rst

```diff
@@ -0,0 +1,23 @@
+Upgrading Mattermost Server with a script
```

Comment:
```suggestion
Upgrading Mattermost Server With a Script
```

Reply:
Why is with With?

Reply:
Ah that was meant to be "with", apologies. I was comparing heading casing and didn't go back to rectify this one.